apps/meteor/server/settings/omnichannel.ts

```diff
@@ -632,6 +632,21 @@ export const createOmniSettings = () =>
 			i18nLabel: 'Call_provider',
 			enableQuery: omnichannelEnabledQuery,
 		});
+
+		await this.add('Livechat_hide_system_messages', ['uj', 'ul', 'livechat-close', 'livechat_webrtc_video_call'], {
```

Comment:
Curious: the other settings for hiding the logo were EE, shouldn't this one be EE too?